rules/windows/registry/registry_set/registry_set_office_trusted_location_uncommon.yml

```diff
@@ -0,0 +1,47 @@
+title: Uncommon Microsoft Office Trusted Location Added
+id: f742bde7-9528-42e5-bd82-84f51a8387d2
+related:
+    - id: a0bed973-45fa-4625-adb5-6ecdf9be70ac
+      type: derived
+status: experimental
+description: Detects changes to the registry keys related to "Trusted Location" of Microsoft Office where the path is set to something uncommon. Attacker might add additional trusted locations to avoid macro security restrictions.
```

Comment:
```suggestion
description: Detects changes to registry keys related to "Trusted Location" of Microsoft Office where the path is set to something uncommon. Attackers might add additional trusted locations to avoid macro security restrictions.
```